_inc/lib/admin-pages/class.jetpack-landing-page.php

```diff
@@ -38,7 +38,7 @@ function get_page_hook() {
 
 	function add_page_actions( $hook ) {
 		// Add landing page specific underscore templates
-		add_action( "admin_footer-$hook",        array( $this, 'js_templates' ) );
+		add_action( "admin_footer-$hook", array( 'Jetpack_Landing_Page', 'js_templates' ) );
```

Comment:
I might suggest referring to it as `__CLASS__` instead, just for clarity, but this way works too.